lib/ansible/modules/net_tools/ldap/ldap_attr.py

```diff
@@ -101,6 +101,13 @@
       - The value(s) to add or remove. This can be a string or a list of
         strings. The complex argument format is required in order to pass
         a list of strings (see examples).
+  validate_cert:
```

Comment:
This should be `validate_certs` as per other modules (e.g. `get_url`). The same anywhere else in the code.